lib/at_monitor/at_monitor.c

```diff
@@ -20,11 +20,22 @@ struct at_notif_fifo {
 	char data[]; /* Null-terminated AT notification string */
 };
 
+/* Workqueue task to dispatch notifications */
 static void at_monitor_task(struct k_work *work);
+static K_WORK_DEFINE(at_monitor_work, at_monitor_task);
+
+/* Workqueue to dispatch notifications from */
+#if CONFIG_AT_MONITOR_USE_DEDICATED_WORKQUEUE
```

Comment:
I'm seeing people mostly using `#if defined`. Is there a specific reason for using plain `#if`?